tests/pinkopy/test_helper.py

```diff
@@ -0,0 +1,52 @@
+import requests_mock
+import uuid
+
+
+def get_uuid():
+    return str(uuid.uuid4())
```

Comment:
I agree on this one. I thought about renaming it to 'get_random_string' so that the implementation could be changed down the line to something other than an UUID but I realized I was overthinking. I'll get rid of get_uuid()